.github/workflows/base-test-image.yaml

```diff
@@ -0,0 +1,21 @@
+name: Build and Push Base Test Image
+
+# trigger manually
+on: workflow_dispatch
+
+jobs:
+  base-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v4
+
+      - name: Build and Push Base Image
```

Comment:
Why not fedora?

Reply:
I don't see Fedora in available runners hosted by GitHub.
https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories